magefiles/utils.go

```diff
@@ -441,8 +441,8 @@ func HandleErrorWithAlert(err error, errLevel slack.ErrorSeverityLevel) error {
 	return nil
 }
 
-func getChangedFiles() (rulesengine.Files, error) {
-	var gitDiff = sh.OutCmd("git", "diff")
+func getChangedFiles(repo string) (rulesengine.Files, error) {
+	var gitDiff = sh.OutCmd(fmt.Sprintf("git -C ../%s", repo), "diff")
```

Comment:
Did you had a chance to test this? I think it won't work either, because [infra-deployments is being cloned to `e2e-tests/tmp/` dir](https://github.com/konflux-ci/e2e-tests/blob/e51393f156f418cdb2e5997b1ced6193e369c717/magefiles/installation/install.go#L107), so either we change that or add a condition here for updating the path

Reply:
fixed